pkg/lib/machine-change-auth.html

```diff
@@ -1,93 +1,80 @@
 <div class="modal-header">
-    <button type="button" class="close" data-dismiss="modal"
+    <button type="button" class="close"
         aria-hidden="true">&times;</button>
     <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
 </div>
 
 <div id="machine-change-auth" class="modal-body">
     {{^available}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}.</p>
+    <p translate="yes">Unable to log in to
+    {{#strong}}{{full_address}}{{/strong}}. The host does not
```

Comment:
The "Unable to log in.." part gets repeated at least four times. Is it possible to factor this out into a standalone string, so that translators don't have to repeat this? The string is quite brittle due to all the formatting, and it's easy to translate this inconsistently.

It could be a separate paragraph, or a heading or so?

Reply:
Changing things for translators is one thing, changing the layout is another. And don't they have tools to deal with repeated translations? I guess we can make it it's own `span`, though.

Reply:
Tools like weblate and such at least give you suggestions, so that might help. However, two of the strings use `{{#strong}}{{full_address}}{{/strong}}`, and the other two use `{{#strong}}{{host}}{{/strong}}`.

But anyway, it's not a big deal. I checked cockpit.pot with this PR, and it at least looks correct now (no excess spaces, no unnecessary mentioning of "cockpit", etc.).